server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java

```diff
@@ -30,5 +30,5 @@
 
     Mono<byte[]> readBytes(BlobId blobId);
 
-    InputStream read(BlobId blobId);
+    Mono<InputStream> read(BlobId blobId);
```

Comment:
we decided that public api should not use reactor types, it's supposed to be a Publisher.

Reply:
I'm coherent with the interface, I'll do another PR to fix that.